src/ir/transforms/structural_hash.cpp

```
@@ -313,7 +313,7 @@ class StructuralHasher {
   }
 
   result_type VisitLeafField(const Span& field) {
-    INTERNAL_UNREACHABLE << "structural_hash should not visit Span field";
+    INTERNAL_UNREACHABLE_SPAN(field) << "structural_hash should not visit Span field";
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)